docs/environment.md

```diff
@@ -17,9 +17,9 @@ In order to build the container follow the steps below:
    git clone https://github.com/fobnail/fobnail-sdk.git
    ```
 3. Go to fobnail SDK directory: `cd fobnail-sdk`
-4. To use latest container checkout `rust` branch: `git checkout rust`
-5. Execute `docker build . -f Dockerfile.rust -t 3mdeb/fobnail-rust-sdk`. It
-   will build the container with following software available:
+4. To use latest SDK version checkout `rust` branch: `git checkout rust`
```

Comment:
...the latest SDK...

Reply:
@arturkow2000 Using branches in documentation makes it outdated quickly. I have just merged this. Please skip the checkout part and just use the default branch.

Reply:
@macpijan Done